Match comments between h and c files

gperciva opened the pull request.

cperciva commented on tests/nc-server/simple_server.c:
```diff
@@ -280,9 +280,9 @@ simple_server_shutdown(void * cookie)
- * simple_server(port, nconn_max, shutdown_after, callback, caller_cookie):
- * Run a server which accepts up to ${nconn_max} connections to port ${port}.
- * After receiving a message, call ${callback} and pass it the
+ * simple_server(sockname, nconn_max, shutdown_after, callback, caller_cookie):
```
Normally the parameter names here should match the ones used in the function definition too.

s/sockname/addr/ ?

gperciva commented:
This PR is blocked on the `crypto_entropy 2^16` question, as well as the existing comment.

gperciva commented:
I removed the libcperciva changes, as they will go in a separate "import" PR.  I also fixed the `sockname` / `addr` discrepancy in the header for `tests/nc-server/simple_server.{c,h}`.

Ready for merge, I think.
